File: January 30, 2026.py
def find_pawn_moves(position):
    letter = position[:1]
    number = int(position[1:])
    possibles_positions = []

    if number == 2:
        possibles_positions.append(letter + str(number + 1))
        possibles_positions.append(letter + str(number + 2))
        return possibles_positions

    possibles_positions.append(letter + str(number + 1))
    return possibles_positions

File: January 30, 2026.js
function findPawnMoves(position) {

  const letter = position.slice(0,1);
  const number = Number(position.slice(1,2));
  let possiblePositions = [];
  
  if (number == 2) {
    possiblePositions.push(letter + (number + 1), letter + (number + 2))
    return possiblePositions
  }

  possiblePositions.push(letter + (number + 1))

  return possiblePositions;
}